src/renderer/components/SchemaDesign/SchemaUtils.js

```diff
@@ -43,18 +43,21 @@ export async function loadMetaTypeInstances(graknTx) {
 }
 
 // attach attribute labels and data types to each node
-export async function computeAttributes(nodes) {
-  return Promise.all(nodes.map(async (node) => {
-    const attributes = await (await node.attributes()).collect();
+export async function computeAttributes(nodes, graknTx) {
+  const nodesComp = await Promise.all(nodes.map(async (node) => {
```

Comment:
I don't think this is any different from returning the promise directly, what's the thinking here?

Reply:
ahh, did that way was I was debugging it. resolving this now ...

Reply:
resolved in https://github.com/graknlabs/workbase/pull/275/commits/306a688e895c28da58bbfe0154b9a5bbc71aaece